packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.scss

```
@@ -53,6 +54,12 @@
   border-bottom: 1px solid $color-yellow-50;
 }
 
+.container-row {
+  border-bottom: 1px solid #e0e0e0;
```

```suggestion
  border-bottom: 1px solid #e0e0e0;
  border-bottom: 1px solid $grey-2;
```
We should use the variables from our color pallette so as to be sure of the uniformity of our colors.